src/components/common/YourCompany/YourCompany.vue

```diff
@@ -179,10 +179,10 @@
       </v-row>
     </div>
 
-    <v-divider v-if="isChangeRegFiling || isFirmConversionFiling" class="mx-4 my-1" />
+    <v-divider v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling" class="mx-4 my-1" />
 
-    <!-- Business Type (alterations, changes and conversions only) -->
-    <div v-if="isAlterationFiling || isChangeRegFiling || isFirmConversionFiling"
+    <!-- Business Type (alterations, changes, conversions and corrections) -->
```

Comment:
Firm corrections. (Not BEN IA corrections.)

Reply:
It is view only and reusable for correction filing. We also need the tooltip from this component.

Reply:
Please add the word "firm" before corrections. Thx!

Reply:
"firm" is now added.